runtime/browser/xwalk_runner.cc

```diff
@@ -144,6 +144,8 @@ void XWalkRunner::OnRenderProcessWillLaunch(content::RenderProcessHost* host) {
   std::vector<extensions::XWalkExtension*> ui_thread_extensions;
   std::vector<extensions::XWalkExtension*> extension_thread_extensions;
 
+// Disable bundled extensions for Linux desktop environment.
```

Comment:
The `!defined(OS_LINUX)` part does not seem to be directly related to disabling the bundling extensions on Linux. Did you forget to provide more context in the commit/pull request message or was this an oversight?